queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/alternate_name.py

```diff
@@ -47,9 +48,13 @@ def update_partner_change(
         alternate_name.change_filing_id = change_filing_rec.id
 
         if start := change_filing.get("filing", {}).get(f"{filing_type}", {}).get("startDate"):
-            start_date = LegislationDatetime.as_utc_timezone_from_legislation_date_str(start)
+            business_start_date = LegislationDatetime.as_utc_timezone_from_legislation_date_str(start)
+            legal_entity.start_date = business_start_date
         else:
-            start_date = alternate_name.start_date
+            business_start_date = alternate_name.business_start_date
+
+        existing_offices = delete_existing_offices(alternate_name)
```

Comment:
GP alternate name records will not have offices attached to it.  Offices will be attached to the LE GP record.